src/components/Explore/Dropdown/Dropdown.style.tsx

```diff
@@ -0,0 +1,73 @@
+import styled from 'styled-components';
+import MuiButton from '@material-ui/core/Button';
+import MuiMenu from '@material-ui/core/Menu';
+import MuiMenuItem from '@material-ui/core/MenuItem';
+import { COLOR_MAP, GREY_5, GREY_0 } from 'common/colors';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+
+export const Button = styled(MuiButton)<{ $open?: boolean; $maxWidth: number }>`
+  border: 1px solid ${GREY_5};
+  border-bottom: ${({ $open }) => $open && 'none'};
+  padding: 0.5rem 0.75rem;
+  border-radius: ${({ $open }) => ($open ? '4px 4px 0 0' : '4px')};
+  width: 100%;
+  letter-spacing: 0;
+
+  &:hover {
+    background-color: ${GREY_0};
+  }
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    max-width: ${({ $maxWidth }) => `${$maxWidth}px`};
+  }
+`;
```

Comment:
Remember to add the focus outline at some point! (and maybe remove the Material UI ripple stuff)